### DISCUSSION TITLE
Custom Discord Avatar with Alert

### DISCUSSION THREAD

**Discussion by sdooweloc** [YoRyan/mailrise#54] [Q&A] [answered]
Is it possible to set a custom Discord Avatar to use with an alert?
I see that you can change the theme depending on if its a .info or .critical or whatever. 
but what I was hoping to do is set like a path to a particular png or something to use when it calls the webhook. Is this possible?

**Comment by YoRyan** [comment, marked as answer]
Mailrise is a direct interface to Apprise, so refer to Apprise's [documentation for the Discord plugin](https://github.com/caronc/apprise/wiki/Notify_discord). Looks like they have avatar and avatar_url parameters that should do what you want.

**Reply by sdooweloc** [reply]
Thanks much
